tests/dslash_test.cpp

```diff
@@ -629,31 +629,15 @@ void dslashRef()
 	tm_matpc(spinorRef->V(), hostGauge, spinor->V(), inv_param.kappa, inv_param.mu, inv_param.twist_flavor, inv_param.matpc_type, dagger, inv_param.cpu_prec, gauge_param);
       else
       {
-        int tm_offset = 12*spinorRef->Volume();
```

Comment:
a note one why I changed this (beyond making things cleaner in my opinion): the `tm_offset` is misleading as it refers to `Volume()`, which suggests that the flavours are separated by an entire space-time-volume. However, the factor `12` is wrong here, it should be `24` (complex) and half the volume instead.